api/admin/routes.py

```diff
@@ -738,6 +738,15 @@ def change_lane_order():
     return app.manager.admin_lanes_controller.change_order()
 
 
+@library_route("/admin/search_field_values", methods=["GET"])
+@has_library
+@returns_json_or_response_or_problem_detail
+@requires_admin
+@requires_csrf_token
```

Comment:
Is csrf token needed here?

Reply:
Seems a bit unnecessary. This call doesn't modify anything, right?

Reply:
You're right! GET requests do not need CSRF tokens, changing...